### ISSUE TITLE
[Enhancement] Option to Toggle Certain In-Game Configs Before Starting Game

### ISSUE THREAD

**Issue by shinra358** [Kaldaien/UnX#31]
Possible to allow to toggle subtitles [on/off], cursor memory, and rumble so that itll be available even when you can't get to the in-game menu yet so every part of the game will always utilize these features?

**Comment by Kaldaien** [comment]
Yes, in theory this can be done. I've not found the memory addresses that govern these features yet.
